concordia/static/js/base.js

```diff
@@ -68,6 +68,33 @@ function displayMessage(level, message, uniqueId) {
     $messages.append($newMessage);
 }
 
+function detectOldIE() {
+    /* 
+        Returns true if user is running IE 11 or older. 
+        Returns false if user is running Edge or non-IE browser.
+    */
+
+    var ua = window.navigator.userAgent;
+    var msie = ua.indexOf('MSIE ');
+    var trident = ua.indexOf('Trident/');
+
+    if (msie > 0 || trident > 0) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+$(function() {
+    if (detectOldIE()) {
+        displayMessage(
+            'danger',
+            'Internet Explorer 11 and older are not fully supported by this website. ' +
+                'Please use a different browser such as Chrome or Firefox.'
```

Comment:
I'm tempted to change this to something like this:

> This website fully supports the current  version of every major browser (Microsoft Edge, Google Chrome, Mozilla Firefox, and Apple Safari). See <a href="https://crowd.loc.gov/help-center/#headingTwelve">our browser support policy</a> for more information and ongoing development efforts.

Reply:
Basically the change I'd most want to make would be not singling out a single browser and giving them a link for more information.

Reply:
HTML in the message isn't working (without further changes) but I changed the wording so it directs them where to go.